app/inputs/multi_value_ordered_input.rb

```diff
@@ -52,14 +52,23 @@ def build_field(value, index)
     end
 
     if value.is_a?(NestedOrderedCreator)
-      creator_options = build_creator_options(value, index)
+      creator_options = build_item_options(value, index, :creator)
       input_creator = @builder.text_field(:creator, creator_options)
     elsif value.is_a?(NestedOrderedTitle)
-      creator_options = build_title_options(value, index)
+      creator_options = build_item_options(value, index, :title)
```

Comment:
creator_options as a variable name doesn't make sense here and in following lines